exercises/tom_clayton/ex02_oo_basics/src/ShapeSorter.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "Shape.h"
+#include <vector>
+#include <memory>
+
+class ShapeSorter
+{
+
+  typedef std::vector<std::shared_ptr<Shape>>::size_type VecSz;
+
+  private: 
+    std::vector<std::shared_ptr<Shape>> m_shpVect;
+    VecSz m_vecLen;
+    
+  public:
+    ShapeSorter(std::vector<std::shared_ptr<Shape>> shpVect);
+
+    const void printShapesType(std::string);
+
+    const void printShapesSides(int);

```

Comment:
For readability it is good practice to have parameter names in the header so people don't have to go to the cpp or docs to figure out what each of the parameters are. e.g. `const void printShapesSides(int sides);`